interfaces/builtin/basedeclaration.go

```diff
@@ -409,6 +409,11 @@ slots:
     allow-installation:
       slot-snap-type:
         - core
+  password:
```

Comment:
As mentioned in the comments, I like 'accounts-control', but also like users-control. Please don't change (or merge! :) until @niemeyer weighs in.